wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -357,21 +357,19 @@ Section headings must not skip a heading level (i.e. do not go from a level 2 he
 
 It is possible to redefine a section's identifier, which is used for linking to it directly. Custom identifiers should be used in case the automatically generated ones are too long or contain tricky punctuation marks or images:
 
-<!-- TODO: remove the {id=...} example when https://github.com/ppy/osu-web/issues/8057 is closed -->
-
 ```markdown
 ## My cooldown has passed. How do I appeal? {#appeal}
 
-## Common restriction reasons and cooldowns {#common-reasons}
-
-## Ideas for a multiplayer match {id=идеи-для-мультиплеера} <!-- alternative syntax for Unicode identifiers -->
+<!-- Alternative form, in case the identifier contains anything except letters, numbers, underscores, and dashes -->
+## Various examples of osu! gameplay {id=osu!-gameplay}
```

Comment:
could keep the unicode text and put the comment on the same line as the heading, since markdown will scream at you if you actually did this

Reply:
it will not, it's a code block

Reply:
yes, so? it's an explanatory example, and appending the comment to the heading will make it less readable due to horizontal scroll. I think it's more important than trying to save a hypothetical someone who will take this exact snippet and reuse it as-is

(besides it's not relevant anymore)